test/unit/destroying_test.rb

```diff
@@ -11,14 +11,29 @@ class CallbackTest < TestResource
   end
 
   def test_destroy
-    stub_request(:delete, "http://example.com/users/6")
+    stub_request(:get, "http://example.com/users/1")
+      .to_return(headers: {content_type: "application/vnd.api+json"}, body: {
+        data: [
+          {id: 1, attributes: {name: "Jeff Ching", email_address: "ching.jeff@gmail.com"}}
+        ]
+      }.to_json)
+
+    users = User.find(1)
+    user = users.first
+    assert(user.persisted?)
+    assert_equal(false, user.new_record?)
+    assert_equal(false, user.destroyed?)
```

Comment:
You can also do `refute user.new_record?`, but it really doesn't matter

Reply:
We can change it if you'd like; we were just trying to follow the style of the existing tests. Do you want us to update?

Reply:
Hi :wave:, recently I played with Rubocop. It suggests to use ActiveSupport method [assert_not](https://apidock.com/rails/ActiveSupport/Testing/Assertions/assert_not)

```ruby
assert_not user.new_record?
```

Reply:
@hashrocketeer, nah. Thanks again for the contribution! I'll push out a new version with your code by eod.